src/ExampleModule.js

```diff
@@ -2,8 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-module.exports = {
-    initialize() {
-        console.log('example module initialized.');
-    },
-  };
\ No newline at end of file
+export function initialize() {
+    console.log('example module initialized.');
```

Comment:
uber nit: Might be nice to decide between single or double quotes. In this PR you use both :)

ESlint has a rule for that which is even auto-fixable: https://eslint.org/docs/rules/quotes.

Reply:
Thanks. I'll fix this here and talk to the team about other defaults.